feature/office/build.gradle

```diff
@@ -37,5 +37,10 @@ dependencies {
     implementation libs.hiltAndroid
     implementation libs.bundles.hilt
     implementation libs.bundles.compose
+    implementation libs.bundles.accompanist
```

Comment:
Remove this because it is not part of your task

Reply:
removing...